test/integration/main_test.go

```diff
@@ -375,27 +394,20 @@ func queryPmapEvents(ctx context.Context, query *bigquery.Query) ([]*v1alpha1.Pm
 	if err != nil {
 		return nil, retry.RetryableError(fmt.Errorf("failed to read job: %w", err))
 	}
-	var pmapEvents []*v1alpha1.PmapEvent
+
+	var entries []*BQRow
 	for {
-		var row []bigquery.Value
+		var row BQRow
```

Comment:
Why remove []bigquery.Value here? What error will it throw if BQ can't unmarshal the bq row to the BQRow type you defined?